.tool-versions

```diff
@@ -0,0 +1 @@
+elixir 1.13
```

Comment:
are we also going to add otp and node version here?

Reply:
No, that file was created because I was using asdf for elixir version control